.github/workflows/admin-jest.yml

```diff
@@ -2,91 +2,66 @@ name: Jest
       codecovToken:
-        description: 'Codecov token'
+        description: "Codecov token"
         required: false
 
+env:
+  COMPOSER_ROOT_VERSION: 6.6.9999999-dev
+
 jobs:
-    run:
-        name: Jest
-        runs-on: ubuntu-latest
-        steps:
-            - uses: shopware/github-actions/shopware-version@main
-              with:
-                fallback: ${{ inputs.shopwareVersionFallback }}
-              id: version
-              if: ${{ inputs.shopwareVersion == '.auto' }}
-            - name: Setup Shopware
-              uses: shopware/setup-shopware@main
-              with:
-                shopware-version:  ${{ steps.version.outputs.shopware-version || inputs.shopwareVersion }}
-                php-version: 8.2
-                env: prod
-                install: 'true'
-    
-            - name: Clone Extension
-              uses: actions/checkout@v4
-              with:
-                path: custom/plugins/${{ inputs.extensionName }}
-        
-            - name: Refresh Plugins
-              run: php bin/console plugin:refresh
-        
-            - name: Install Plugin
-              run: php bin/console plugin:install --activate ${{ inputs.extensionName }}
-        
-            - name: Setup Node
-              uses: actions/setup-node@v4
-              with:
-                node-version: '20.x'
-        
-            - name: Install Administration dependencies
-              working-directory: src/Administration/Resources/app/administration
-              run: npm install --no-audit --no-fund --prefer-offline
-        
-            - name: Prepare Jest environment
-              working-directory: src/Administration/Resources/app/administration
-              run: npm run unit-setup
-        
-            - name: Generate Entity Schema
-              run: php bin/console framework:schema -s 'entity-schema' src/Administration/Resources/app/administration/test/_mocks_/entity-schema.json
-        
-            - name: Install Plugin Dependencies
-              working-directory: custom/plugins/${{ inputs.extensionName }}/src/Resources/app/administration
-              run: npm install --no-audit --no-fund --prefer-offline
-        
-            - name: Jest Unit Tests
-              working-directory: custom/plugins/${{ inputs.extensionName }}/src/Resources/app/administration
-              run: npm run unit -- --coverage
+  run:
+    name: Jest
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Extension
+        uses: shopware/github-actions/setup-extension@main
+        with:
+          extensionName: ${{ github.event.repository.name }}
+          install: true
+          installAdmin: true
+          env: prod
+          extraRepositories: |
+            {
+              "${{ github.event.repository.name }}": {
+                "type": "path",
+                "url": "custom/plugins/${{ github.event.repository.name }}",
+                "symlink": true
+              }
+            }
```

Comment:
This should not be necessary, IMHO. We have all the information in the action to do this automatically

Reply:
Will be part of another refactor of `setup-extension`.